packages/web/src/components/unstake/unstake-liquidity/UnstakeLiquidity.stories.tsx

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import { ComponentStory, ComponentMeta } from "@storybook/react";
+import UnstakeLiquidity from "./UnstakeLiquidity";
+import UnstakeLiquidityContainer from "@containers/unstake-liquidity-container/UnstakeLiquidityContainer";
+
+export default {
+  title: "unstake/UnstakeLiquidity",
+  component: UnstakeLiquidity,
+} as ComponentMeta<typeof UnstakeLiquidity>;
+
+const Template: ComponentStory<typeof UnstakeLiquidity> = () => {
+  return <UnstakeLiquidityContainer />;
```

Comment:
I don't understand the use of UnstakeLiquidityContainer here.

Reply:
@luvhaeun 
UnstakeLiquidityContainer was used to get the same function and state values used by the UnstakeLiquidityContainer.
Is there a better way??

Reply:
@hjlee0 This story is about the UnstakeLiquidity component. You do not need to create a story for the UnstakeLiquidityContainer.